ironmon_tracker/data/PokemonData.lua

```diff
@@ -1855,7 +1862,8 @@ PokemonData.Pokemon = {
 		evolution = PokemonData.Evolutions.NONE,
 		bst = "410",
 		movelvls = { { 3, 6, 10, 15, 24, 34, 45, 57 }, { 3, 6, 10, 15, 24, 34, 45, 57 } },
-		weight = 28.5
+		weight = 28.5,
+		friendshipbase = 70 -- setting important default base friendships for Friend tracking -Tainted_Wolf
```

Comment:
You probably meant to add this to Azurill and not Azumarill, as the former is the one that needs this and is missing it

Reply:
Yup, I was copy/pasting from a local copy of the tracker that was very outdated, and I made a mistake here. Thanks for catching it! Fixed this pretty easy.